conan/tools/meson/meson.py

```diff
@@ -112,6 +114,15 @@ def _install_verbosity(self):
         verbosity = self._conanfile.conf.get("tools.build:verbosity", choices=("quiet", "verbose"))
         return "--quiet" if verbosity else ""
 
+    @property
+    def _install_strip(self) -> str:
```

Comment:
No need to make a property for just 1 private thing that is used just once in the code, and can be done in 1 line

Reply:
Done [b2bc13c](https://github.com/conan-io/conan/pull/18429/commits/b2bc13ceb345ee274f223d1d33ec04fc1ace7057)